benchexec/tools/coveriteam-verifier-validator.py

```diff
@@ -50,14 +51,17 @@ def determine_result(self, run):
         will print out the produced aftifacts.
         If more than one dict is printed, the first matching one.
         """
-        for line in run.output:
+        verdict = None
+        for line in reversed(run.output):
             line = line.strip()
-            if "verdict" in line:
+            if "'verdict':" in line and verdict is None:
                 # CoVeriTeam outputs benchexec result categories as verdicts.
                 try:
-                    d = ast.literal_eval(line)
-                    if isinstance(d, dict):
-                        return d.get("verdict", result.RESULT_ERROR)
-                except SyntaxError:
+                    verdict = re.search(r"'verdict': '([a-zA-Z\(\)\ ]*)'", line).group(1).lower()
+                except:
```

Comment:
What kind of exceptions are expected here?

Reply:
Oh man, sorry, that was very error-prone.
I added the expected `AttributeError` from the `.group(1)` call in 0eb8a6a for documentation purposes, but it is gone in the latest version.